heat_pump_adoption_modelling/pipeline/MCS/mcs_epc_joining.py

```diff
@@ -0,0 +1,322 @@
+# File: heat_pump_adoption_modelling/pipeline/MCS/mcs_epc_joining.py
+"""Joining the MCS and EPC datasets."""
+
+import pandas as pd
+import numpy as np
+import string
+import re
+import recordlinkage as rl
```

Comment:
Make sure the dependencies are all in requirements.txt

Reply:
Added, thanks!